services/api/src/utils/schema.js

```diff
@@ -85,6 +85,20 @@ function createSchema(attributes = {}, options = {}) {
     return this.save();
   };
 
+  schema.post(/^init|save/, function () {
+    if (!this.$locals.auditOriginal) {
+      this.$locals.auditOriginal = this.toObject({
```

Comment:
Can we just call this `original` as this has nothing specific to do with an audit

Reply:
we sure can